comms/discovery/schema/schema.go

```diff
@@ -201,25 +225,26 @@ type Summary struct {
 }
 
 type ChatWebsocketEventData struct {
-	Metadata Metadata   `json:"metadata"`
-	RPC      RPCPayload `json:"rpc"`
+	Metadata Metadata `json:"metadata"`
+	RPC      RPC      `json:"rpc"`
 }
 
 type Metadata struct {
 	Timestamp string `json:"timestamp"`
 	UserID    string `json:"userId"`
 }
 
-type RPCPayload struct {
-	Method    RPCMethod        `json:"method"`
-	Params    RPCPayloadParams `json:"params"`
-	Timestamp float64          `json:"timestamp"`
+type RPC struct {
```

Comment:
aw man, we got like redundant stuff now? I'm not sure where this came from or why quicktype wasn't getting it before but it's identical to `RPCPayload`

Reply:
yeah this confused me as well... it seems to have introduced a new intermediate type.  Maybe a quicktype change?

Or it is possible that there was some drift between the two `.ts` files... this PR does ensure we always use `severTypes.ts`.  Will check on that.

Reply:
fixed by pinning quicktype version